<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCircuito</class>
 <widget class="QMainWindow" name="MainCircuito">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de Circuitos Digitais</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_cir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>385</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>CIRCUITOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tv">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>0</y>
      <width>385</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TABELA VERDADE</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>PORTAS E CONEXÃ•ES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_saidas">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>SAIDAS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tab_port">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>291</width>
      <height>525</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>TIPO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> #
INPUT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> IN
 #1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> IN
 #2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> IN
 #3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> IN
 #4</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tab_saidas">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>95</width>
      <height>525</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>48</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>OUT
FROM</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tab_tv">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>30</y>
      <width>385</width>
      <height>555</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCircuito">
    <property name="title">
     <string>Circuito</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionLer"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuSimular">
    <property name="title">
     <string>Simular</string>
    </property>
    <addaction name="actionGerar_Tabela"/>
   </widget>
   <addaction name="menuCircuito"/>
   <addaction name="menuSimular"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo...</string>
   </property>
  </action>
  <action name="actionLer">
   <property name="text">
    <string>Ler...</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar...</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionGerar_Tabela">
   <property name="text">
    <string>Gerar Tabela</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
